pytorch_lightning/utilities/distributed.py

```diff
@@ -381,6 +389,18 @@ def init_dist_connection(
     world_size = world_size if world_size is not None else cluster_environment.world_size()
     os.environ["MASTER_ADDR"] = cluster_environment.master_address()
     os.environ["MASTER_PORT"] = str(cluster_environment.master_port())
+
+    # local rank mapping for device open is needed for hpu devices
```

Comment:
Could this be done directly within the HPU Accelerator ?

Reply:
I dont see a way to do it.